llvm/lib/Target/AIE/AIEHazardRecognizer.h

```diff
@@ -26,6 +26,7 @@
 #include "llvm/MC/MCInstrItineraries.h"
 
 namespace llvm {
+using MemoryBankBits = uint64_t;
```

Comment:
I think that, as everything is handled as 32bit (bitmask, `AIEHazardRecognizer::getMemoryBanks`), for consistency we can use `uint32_t`, or just `unsigned`, as `getMemoryBanks`.